.github/workflows/testing.yml

```diff
@@ -114,6 +117,7 @@ jobs:
         export PYPI_TOKENS=$(mktemp)
         cat > ${PYPI_TOKENS} <<EOF
         examples/shouldi=${{ secrets.PYPI_SHOULDI }}
+        examples/ffmpeg=${{ secrets.PYPI_EXAMPLES_FFMPEG }}
```

Comment:
```suggestion
```

Thanks for adding these! However I don't think we need to add this one, since we only add lines here it we'll be publishing a package, and I don't think we need to publish one since we're having them write it.